tests/test_cache_lock_file_perf.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import multiprocessing
+import os
+import time
+
+from portalocker import exceptions
+import pytest
+
+from msal_extensions import FilePersistence, CrossPlatLock
+
+
+@pytest.fixture
+def cache_location():
+    path_to_script = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(path_to_script, str(os.getpid())+"msal.cache")
```

Comment:
1. Taking a close look, does the `os.getpid()` work as expected? Wouldn't it always have same PID among our 2+ test cases here, thus unintentionally share same file among all the test cases? While that does not currently seem to make a difference in terms of test output, the 3 test cases still unnecessarily couple with each other.

   Perhaps you can take a look into Python's built-in tempfile module.

2. Also, let's follow `test_persistence.py` approach to close the test file by this fixture, otherwise there seems no point to use the fixture feature to begin with.

Reply:
I can just use the temp_file fixture we have used in test_persistence.py, that seems to be working fine as well. Will update it in the next commit